### ISSUE TITLE
Como proceder quando alguém viola o código de conduta?

### ISSUE THREAD

**Issue by soapdog** [NiteroiJS/code-of-conduct#3]
Essa discussão visa criar um SOP para mostrar quais procedimentos são necessários quando alguém viola o código de conduta.

**Comment by vagnerzampieri** [comment]
Tem que depender do nível de violação e se constitue crime ou não. Se é crime tem que ser identificados os envolvidos, provas, testemunhas e levar o caso para a polícia resolver. Se é código de conduta apenas, seria melhor fazer um tópico para punições e nível. Precisa ter muito cuidado quando for divulgar, estamos lidando com vidas reais, ser divulgado de forma irresponsável pode ter consequências graves. Poderia ter um usuário da organização, sem nome para notificar esse tipo de coisa, sem estar atrelado a ninguém, para não dar margem a fofoca. Antes de fazer, precisa ter provas claras sobre o ocorrido, se for de consentimento, ambos precisam ser punidos.

**Comment by leobalter** [comment]
além do que foi falado, é interessante ver como é possível educar a comunidade para que não haja recorrência, após observar todos os valores já citados, etc.

**Comment by vagnerzampieri** [comment]
Para educar, precisa ser falado no começo de cada encontro, como um assunto introdutório, igual a FIFA faz em jogos importantes da copa, em que os capitães fazem um discurso e tal. É meio chato isso, mas se quiserem educar, é uma boa forma.

**Comment by leobalter** [comment]
concordo, mas acho que isso é apenas uma medida preventiva, não seria interessante ter uma questão a ser tratada pós incidente?

Além disso, o código de conduta é para todo o grupo, em todos os momentos, não apenas em eventos.

**Comment by vagnerzampieri** [comment]
Sim, falei referente a como divulgar em eventos, e quando digo eventos é aula, encontro de grupo, grandes eventos, qualquer coisa que reuna pessoa e que defendam essa mudança. Para caso grupos de rede sociais, é a simples divulgação do código de tempos em tempos, tem outras coisas em mente?

**Comment by leobalter** [comment]
só o que já falei mesmo, que é o pós-incidente.

**Comment by vagnerzampieri** [comment]
Uhm, entendi, abriram essa aqui https://github.com/NiteroiJS/code-of-conduct/issues/8
